src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -1596,36 +1627,30 @@ SubCmdValidate::execute(const SubCmdOptions& _options) const
     if (testsToRun.empty())
       throw std::runtime_error("No test given to validate against.");
 
-    // Examine test entries
-    for (const auto &userTestName : testsToRun) {
-      const std::string userTestNameLC = boost::algorithm::to_lower_copy(userTestName);   // Lower case the string entry
+    // Validate the user test requests
+    for (auto &userTestName : testsToRun) {
+      userTestName = get_run_test_name(userTestName);
```

Comment:
this works? have you tested `xbutil examine -r quick` and `-r all`?

Reply:
It does. Tested with `xbutil validate -r quick` and `xbutil validate -r all`. The function `get_run_test_name` issues the deprecation warning and puts all characters into their lowercase form. If no warning is issued then whatever test the user passed in is returned (in lowercase)

Reply:
Quick test
```
dbenusov@xsjpranjal01:/proj/xsjhdstaff6/dbenusov/XRT$ xbutil validate -r quick -d 02:00
Starting validation for 1 devices

Validate Device           : [0000:02:00.1]
    Platform              : xilinx_u280_xdma_201920_3
    SC Version            : 4.3.15
    Platform ID           : 0x5e278820
-------------------------------------------------------------------------------
Test 1 [0000:02:00.1]     : aux-connection
    Test Status           : [PASSED]
-------------------------------------------------------------------------------
Test 2 [0000:02:00.1]     : pcie-link
    Warning(s)            : Link is active
                            Please make sure that the device is plugged into Gen 3x16,
                            instead of Gen 3x1. Lower performance maybe experienced.
    Test Status           : [PASSED WITH WARNINGS]
-------------------------------------------------------------------------------
Test 3 [0000:02:00.1]     : sc-version
    Test Status           : [PASSED]
-------------------------------------------------------------------------------
Test 4 [0000:02:00.1]     : verify
    Test Status           : [PASSED]
-------------------------------------------------------------------------------
Validation completed, but with warnings. Please run the command '--verbose' option for more details
```
All test
```
dbenusov@xsjpranjal01:/proj/xsjhdstaff6/dbenusov/XRT$ xbutil validate -r all -d 02:00
Starting validation for 1 devices

Validate Device           : [0000:02:00.1]
    Platform              : xilinx_u280_xdma_201920_3
    SC Version            : 4.3.15
    Platform ID           : 0x5e278820
-------------------------------------------------------------------------------
Test 1 [0000:02:00.1]     : aux-connection
    Test Status           : [PASSED]
-------------------------------------------------------------------------------
Test 2 [0000:02:00.1]     : pcie-link
    Warning(s)            : Link is active
                            Please make sure that the device is plugged into Gen 3x16,
                            instead of Gen 3x1. Lower performance maybe experienced.
    Test Status           : [PASSED WITH WARNINGS]
-------------------------------------------------------------------------------
Test 3 [0000:02:00.1]     : sc-version
    Test Status           : [PASSED]
-------------------------------------------------------------------------------
Test 4 [0000:02:00.1]     : verify
    Test Status           : [PASSED]
-------------------------------------------------------------------------------
Test 5 [0000:02:00.1]     : dma
    Details               : Host -> PCIe -> FPGA write bandwidth = 779.1 MB/s
                            Host <- PCIe <- FPGA read bandwidth = 738.8 MB/s
    Test Status           : [PASSED]
-------------------------------------------------------------------------------
Test 6 [0000:02:00.1]     : iops
    Details               : IOPS: 81929 (hello)
    Test Status           : [PASSED]
-------------------------------------------------------------------------------
Test 7 [0000:02:00.1]     : mem-bw
    Details               : Maximum throughput: 43748 MB/s
    Test Status           : [PASSED]
-------------------------------------------------------------------------------
Test 8 [0000:02:00.1]     : vcu
Validation completed, but with warnings. Please run the command '--verbose' option for more details
```